Makefile

```
@@ -93,6 +93,13 @@ test_fast::
 test_all::
 	$(GO_TEST) $(TESTABLE_PKGS)
 
+generate_schema:: $(OPENAPI_FILE)
+	$(call STEP_MESSAGE)
+	$(GO) install $(VERSION_FLAGS) $(PROJECT)/cmd/$(CODEGEN)
+	echo "Generating Pulumi schema..."
+	$(CODEGEN) schema $(OPENAPI_FILE) pkg/gen/$${LANGUAGE}-templates $(PACKDIR)
```

Actually, I think the `$${LANGUAGE}` part is wrong.

---

Yeah, you can omit that argument.
```suggestion
	$(CODEGEN) schema $(OPENAPI_FILE) "" $(PACKDIR)
```